### PR TITLE
Add retry

### PR DESCRIPTION
**Pull request by ZsoltPath**

### CODE REVIEW

**Comment by ZsoltPath** [comment]
@vaunus 
https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Config.html

File: packages/cloudfront_invalidate/index.ts [hunk 1]
```diff
@@ -29,6 +30,10 @@ const run = async (): Promise<void> => {
       },
     };
 
+    // @ts-ignore
+    const config = new AWS.Config({
+      maxRetries: 5,
+    });
     const cloudfront = new CloudFront({ apiVersion: '2019-03-26', customUserAgent: 'aws-github-actions-cloudfront' });
```
Comment (vaunus) [inline packages/cloudfront_invalidate/index.ts]:
```suggestion
    const cloudfront = new CloudFront({ apiVersion: '2019-03-26', customUserAgent: 'aws-github-actions-cloudfront', maxRetries: 5 });
```

File: packages/cloudfront_invalidate/index.ts [hunk 2]
```diff
@@ -29,6 +30,10 @@ const run = async (): Promise<void> => {
       },
     };
 
+    // @ts-ignore
+    const config = new AWS.Config({
+      maxRetries: 5,
+    });
```
Comment (vaunus) [inline packages/cloudfront_invalidate/index.ts]:
```suggestion
```

File: packages/cloudfront_invalidate/index.ts [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import * as core from '@actions/core';
 import * as CloudFront from 'aws-sdk/clients/cloudfront';
+import * as AWS from 'aws-sdk';
```
Comment (vaunus) [inline packages/cloudfront_invalidate/index.ts]:
```suggestion
```

**Review by vaunus — APPROVED** [review]

**Pull request merged**